.editorconfig

```diff
@@ -49,11 +49,11 @@ end_of_line = crlf
 # Organize usings
 dotnet_sort_system_directives_first = true
 
-# Use this.
-dotnet_style_qualification_for_field = true:warning
-dotnet_style_qualification_for_property = true:warning
-dotnet_style_qualification_for_method = true:warning
-dotnet_style_qualification_for_event = true:warning
+# Avoid this., reduce the code size and complexity to make it irrelevant instead

```

Comment:
Hopefully self-explanatory. This used to be one of the unfortunate styles forced on everyone by the misguided StyleCop rules 20 years ago.